src/seedu/addressbook/data/person/Name.java

```diff
@@ -59,5 +59,9 @@ public boolean equals(Object other) {
     public int hashCode() {
         return fullName.hashCode();
     }
-
+    
+    @Override
+	public String getPrintableString() {
+		return "Name:"+fullName;
```

Comment:
Notice that your indentation mechanism uses tabs for indentation. 
The coding standards we follow require using 4 spaces for indentation.
Tip: you can set Eclipse to do this for you, check out how!